### PR TITLE
Add wav2vec2 TPU tests

### PR DESCRIPTION
**Pull request by taylanbil**

### CODE REVIEW

File: tests/pytorch/nightly/wav2vec2.libsonnet [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      datasets: common.datasetsVolume,
+    },
+    jobSpec+:: {
+      template+: {
+        spec+: {
+          containerMap+: {
+            train+: {
+              resources+: {
+                requests: {
+                  cpu: "40.0",
```
Comment (zcain117) [inline tests/pytorch/nightly/wav2vec2.libsonnet:97]: could you try making these top-level like here: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/pytorch/nightly/fs-transformer.libsonnet#L61

and see if it changes the `.yaml` at all?  I think it should translate to the same yaml

**Comment by taylanbil** [comment]
Functional test passed, fired up a conv. test.

cc @bilgeacun @alexeib

**Review by zcain117 — APPROVED** [review]
LGTM assuming the oneshots pass

**Comment by taylanbil** [comment]
Convergence oneshot test passed.

**Pull request merged**